.github/workflows/main.yml

```diff
@@ -225,6 +226,8 @@ jobs:
   build-graalvm-windows:
     name: /substratevm on Windows
     runs-on: windows-2022
+    matrix: 
+      os: [windows-2022, windows-2019]
```

Comment:
The matrix is not used. Also, Native Image needs Visual Studio 2022 or later, so it won't build on `windows-2019`.

Reply:
Understood! I'll take it out